apps/meteor/client/views/omnichannel/departments/NewDepartment.tsx

```diff
@@ -23,13 +23,12 @@ const NewDepartment = ({ id, reload, refetchRef }: NewDepartmentProps) => {
 		refetchRef.current = refetch;
 	}, [refetchRef, refetch]);
 
-	const isMultipleDepartmentsAvailable = hasLicense || (!isLoading && data?.total !== undefined && data.total < 1);
 	const t = useTranslation();
 
-	if (isMultipleDepartmentsAvailable === 'loading' || undefined) {
+	if (isLoading || hasLicense === 'loading') {
 		return <PageSkeleton />;
 	}
-	if (!isMultipleDepartmentsAvailable) {
+	if (hasLicense === false || (data?.total !== undefined && data.total < 1)) {
```

Comment:
At this point `hasLicence` is a boolean, right? If so `!hasLicence` is enough. 
Also for the second part `data?.total < 1` or `data?.total === 0` is enough, since `undefined < 1` or `undefined === 0` are both `false`.